Nasal/engine.nas

```diff
@@ -316,6 +316,27 @@ controls.throttleMouse = func {
     setprop("/controls/engines/current-engine/throttle", new_value);
 };
 
+# 2018.2 introduces new "all" properties for throttle, mixture and prop pitch.
+# this is the correct way to interface with the axis based controls - use a listener
+# on the *-all property
+_setlistener("/controls/engines/throttle-all", func{
```

Comment:
Why are you using `_setlistener` instead of `setlistener`? The [wiki](http://wiki.flightgear.org/Using_listeners_and_signals_with_Nasal#setlistener.28.29_vs._setlistener.28.29) says `setlistener` should be used.

Reply:
I had run time errors because setlistener wasn't defined; I suspect it is the inclusion order - so I followed the example in other FGdata/Nasal modules

Reply:
There are multiple `setlistener` calls already in the file so it would be strange to get runtime errors.

Reply:
sorry my mistake; I copied the code from controls.nas and obviously it doesn't need to be a _setlistener.